server/app.js

```diff
@@ -47,6 +50,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/notification", notificationRouter);
```

Comment:
```suggestion
app.use("/notifications", notificationRouter);
```
Nit: Naming our routes as per REST guidelines